src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -70,7 +70,7 @@ function SidebarLinks({onLinkClick, insets, optionListItems, isLoading, priority
         const unsubscribeOnyxModal = onyxSubscribe({
             key: ONYXKEYS.MODAL,
             callback: (modalArg) => {
-                if (_.isNull(modalArg)) {
+                if (_.isNull(modalArg) || _.isUndefined(modalArg)) {
```

Comment:
Maybe it would be better to check if `modalArg` is **not what we are looking for** instead of checking if it's **something we are not looking for** (null, undefined, etc).

Reply:
did you mean it like this?

Maybe it would be better to check if modalArg is **what we are looking for** instead of checking if it's **something we are not looking for** (null, undefined, etc).

If so, what would it be that we're looking for. I think it's better to just early-return in case of nullish values.

If not, what do you mean by that?

Reply:
I meant something like - is it supposed to be a string? If so, then neither `null` or `undefined` are `String`

Reply:
Ok, got it! Thanks!

Reply:
In this case, `ONYXKEYS.MODAL` is an object, so we'll still have to check for `null`, because `null` is treated as type `object`